docs/form/vich_file_type.md

```diff
@@ -32,6 +33,33 @@ allow_delete
 ------------
 **type**: `bool` **default**: `true`
 
+delete_label
+--------------
+**type**: `string`, `callable`, `Symfony\Component\PropertyAccess\PropertyPath` **default**: `vich_uploader.form_label.delete_confirm`
+
+Can be string 
+```php
+use Vich\UploaderBundle\Form\Type\VichImageType;
+
+$builder->add('genericFile', VichImageType::class, [
+    'delete_label' => 'delete_file',
+]);
+
+```
+
+Can be callable
```

Comment:
Are you sure label can be a callable?

Reply:
You're right, I quickly supposed that but contrary to _download_label_, it's not.
I'm correcting.

Reply:
Done (also removed Property path option as after testing it wasn't working neither, and improved the example to showcase an actual usage of it)